test_scripts/API/VehicleData/HandsOffSteering/common.lua

```diff
@@ -0,0 +1,285 @@
+---------------------------------------------------------------------------------------------------
+-- Common module
+---------------------------------------------------------------------------------------------------
+---------------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 6
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 2
+config.application2.registerAppInterfaceParams.syncMsgVersion.majorVersion = 6
+config.application2.registerAppInterfaceParams.syncMsgVersion.minorVersion = 2
+
+--[[ Required Shared libraries ]]
+local actions = require("user_modules/sequences/actions")
+local runner = require('user_modules/script_runner')
+local utils = require("user_modules/utils")
+local json = require("modules/json")
+local SDL = require("SDL")
+
+--[[ General configuration parameters ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local m = {}
+local hashId = {}
+
+m.Title = runner.Title
+m.Step = runner.Step
+m.getPreloadedPT = actions.sdl.getPreloadedPT
+m.setPreloadedPT = actions.sdl.setPreloadedPT
+m.registerApp = actions.registerApp
+m.registerAppWOPTU = actions.registerAppWOPTU
+m.activateApp = actions.activateApp
+m.getMobileSession = actions.getMobileSession
+m.getHMIConnection = actions.getHMIConnection
```

Comment:
@TMyza Please avoid to use wrappers from actions module in new scripts. Its purpose is to provide backward compatibility with old scripts only.

```suggestion
m.registerApp = actions.app.register
m.registerAppWOPTU = actions.app.registerNoPTU
m.activateApp = actions.app.activate
m.getMobileSession = actions.mobile.getSession
m.getHMIConnection = actions.hmi.getConnection
```